crates/solvers/src/infra/config/mod.rs

```diff
@@ -1,3 +1,23 @@
+use std::fmt::Debug;
+
 pub mod baseline;
 pub mod dex;
 pub mod legacy;
+
+/// Unwraps result or logs a `TOML` parsing error.
+fn unwrap_or_log<T, E, P>(result: Result<T, E>, path: &P) -> T
```

Comment:
This feels like it should live in the `util/result.rs` or equivalent instead of here. No strong opinion though.